[0m13:39:08.589741 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x110ed34d0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x113a37b10>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x113a379d0>]}


============================== 13:39:08.599342 | 8bf88270-c84b-49e7-a015-d3168cca5b13 ==============================
[0m13:39:08.599342 [info ] [MainThread]: Running with dbt=1.10.5
[0m13:39:08.599812 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'log_cache_events': 'False', 'write_json': 'True', 'partial_parse': 'True', 'cache_selected_only': 'False', 'profiles_dir': '/Users/u249637/.dbt', 'debug': 'False', 'fail_fast': 'False', 'log_path': 'logs', 'version_check': 'True', 'warn_error': 'None', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'empty': 'None', 'warn_error_options': 'WarnErrorOptionsV2(error=[], warn=[], silence=[])', 'invocation_command': 'dbt init olympic_games', 'log_format': 'default', 'static_parser': 'True', 'target_path': 'None', 'introspect': 'True', 'send_anonymous_usage_stats': 'True'}
[0m13:39:08.706533 [debug] [MainThread]: Starter project path: /Users/u249637/Documents/Personal/sophie_practice_sql/database_design_practice/dbt_venv/lib/python3.13/site-packages/dbt/include/starter_project
[0m13:39:08.745856 [info ] [MainThread]: 
Your new dbt project "olympic_games" was created!

For more information on how to configure the profiles.yml file,
please consult the dbt documentation here:

  https://docs.getdbt.com/docs/configure-your-profile

One more thing:

Need help? Don't hesitate to reach out to us via GitHub issues or on Slack:

  https://community.getdbt.com/

Happy modeling!

[0m13:39:08.746263 [info ] [MainThread]: Setting up your profile.
[0m13:39:32.552984 [info ] [MainThread]: Profile olympic_games written to /Users/u249637/.dbt/profiles.yml using target's profile_template.yml and your supplied values. Run 'dbt debug' to validate the connection.
[0m13:39:32.567265 [debug] [MainThread]: Resource report: {"command_name": "init", "command_success": true, "command_wall_clock_time": 24.096092, "process_in_blocks": "0", "process_kernel_time": 0.67455, "process_mem_max_rss": "111771648", "process_out_blocks": "0", "process_user_time": 1.842015}
[0m13:39:32.567888 [debug] [MainThread]: Command `dbt init` succeeded at 13:39:32.567723 after 24.10 seconds
[0m13:39:32.568329 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x113b0e9e0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x1130dd490>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x113b1c8d0>]}
[0m13:39:32.568758 [debug] [MainThread]: Flushing usage events
[0m13:39:32.991706 [debug] [MainThread]: An error was encountered while trying to flush usage events
[0m08:10:48.237733 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x10821f4d0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x10973bb10>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x10973b9d0>]}


============================== 08:10:48.249537 | 31348c1e-f2cb-47f3-bc97-dee671270f05 ==============================
[0m08:10:48.249537 [info ] [MainThread]: Running with dbt=1.10.5
[0m08:10:48.250069 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'write_json': 'True', 'log_cache_events': 'False', 'partial_parse': 'True', 'cache_selected_only': 'False', 'warn_error': 'None', 'version_check': 'True', 'fail_fast': 'False', 'log_path': 'logs', 'debug': 'False', 'profiles_dir': '/Users/u249637/.dbt', 'use_colors': 'True', 'use_experimental_parser': 'False', 'empty': 'False', 'quiet': 'False', 'no_print': 'None', 'warn_error_options': 'WarnErrorOptionsV2(error=[], warn=[], silence=[])', 'static_parser': 'True', 'introspect': 'True', 'log_format': 'default', 'target_path': 'None', 'invocation_command': 'dbt run --full-refresh', 'send_anonymous_usage_stats': 'True'}
[0m08:10:48.251042 [error] [MainThread]: Encountered an error:
Runtime Error
  No dbt_project.yml found at expected path /Users/u249637/Documents/Personal/sophie_practice_sql/database_design_practice/dbt_practice/dbt_project.yml
  Verify that each entry within packages.yml (and their transitive dependencies) contains a file named dbt_project.yml
  
[0m08:10:48.272993 [debug] [MainThread]: Resource report: {"command_name": "run", "command_success": false, "command_wall_clock_time": 0.16332516, "process_in_blocks": "0", "process_kernel_time": 0.855537, "process_mem_max_rss": "110804992", "process_out_blocks": "0", "process_user_time": 1.920525}
[0m08:10:48.273582 [debug] [MainThread]: Command `dbt run` failed at 08:10:48.273472 after 0.16 seconds
[0m08:10:48.273970 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x10972eea0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x1097660f0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x1095e1590>]}
[0m08:10:48.274310 [debug] [MainThread]: Flushing usage events
[0m08:10:48.669472 [debug] [MainThread]: An error was encountered while trying to flush usage events
